canonical-websites/www.ubuntu.com
# removed all the erroring bits of the magnifying glass svg

## Done

* used https://validator.nu/ to trim out bad XML bits of the magnifying glass

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Compare the search icon to live

## Issue / Card

Fixes #1947

## Screenshots

live
![image](https://user-images.githubusercontent.com/441217/27843863-1fd9ee08-6111-11e7-8ecf-a95618b1049a.png)

updated
![image](https://user-images.githubusercontent.com/441217/27843859-0a3c2638-6111-11e7-9390-405020fc65c6.png)
